### PR TITLE
[DOCS][MINOR] Screenshot + minor fixes to improve reading for accumulators

### PR DESCRIPTION
**Pull request by jaceklaskowski**
## What changes were proposed in this pull request?

Added screenshot + minor fixes to improve reading
## How was this patch tested?

Manual

### CODE REVIEW

File: docs/programming-guide.md [hunk 1]
```diff
@@ -1328,12 +1328,18 @@ value of the broadcast variable (e.g. if the variable is shipped to a new node l
...
 therefore be efficiently supported in parallel. They can be used to implement counters (as in
 MapReduce) or sums. Spark natively supports accumulators of numeric types, and programmers
-can add support for new types. If accumulators are created with a name, they will be
+can add support for new types.
+
+If accumulators are created with a name, they will be
 displayed in Spark's UI. This can be useful for understanding the progress of
 running stages (NOTE: this is not yet supported in Python).
 
+<p style="text-align: center;">
```
Comment (srowen) [inline docs/programming-guide.md]: I think this all looks OK. Can this just use standard markdown image syntax or was this HTML required for some reason?

Comment (jaceklaskowski) [inline docs/programming-guide.md]: I copied it from another file https://github.com/apache/spark/blob/master/docs/cluster-overview.md as I didn't know how to include images.

**Comment by HyukjinKwon** [comment]
@jaceklaskowski I think the title is not clear for the changes this PR has.

**Comment by jaceklaskowski** [comment]
@HyukjinKwon In that case I'd ask for the alternative as I currently have no idea how to make it clearer (it wasn't me to say "the title is not clear" :)) What do you miss in the title?

**Comment by HyukjinKwon** [comment]
@jaceklaskowski Because I thought obviously it is not clear. For me it sounds like adding whole documents for Accumulators. 
As you just said, I think "Added screenshot and minor fixes to improve reading for Accumulators in the programming guide" would be better if this is grammatically correct.

**Comment by HyukjinKwon** [comment]
@jaceklaskowski Would you accept my PR if I fix a bug in datasource in Spark SQL and I name it as "datasource"?

**Comment by srowen** [comment]
@jaceklaskowski "Accumulators" is certainly not a complete title. Accumulators .. what? I like @HyukjinKwon 's suggestion. Although it's minor it sets the right example, and good titles everywhere make it easier to browse and find issues later.

File: docs/programming-guide.md (hunk 1 above)
Comment (jodersky) [inline docs/programming-guide.md]: the usual way to include images is `alttext`, in your case `Accumulators in the Spark UI`
This does not include centering though.

**Comment by jaceklaskowski** [comment]
Title has changed to incorporate @HyukjinKwon's suggestion. Please review @srowen. I intentionally have not included "in the programming guide" due to "[DOCS]". Is this ok now?

**Comment by jaceklaskowski** [comment]
Could this PR have a bit of @srowen's attention? :)

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**